lib/tasks/thumbnails.rake

```diff
@@ -64,5 +64,30 @@ namespace :pulmap do
         end
       end
     end
+
+    desc 'Pre-cache thumbnails by Solr query'
+    task :precache_query, %i[query override_existing] => [:environment] do |_t, args|
+      raise 'Please supply required arguments [query]. rake precache_qeury["layer_slug_s:*"]' unless args[:query]
```

Comment:
Small typo. Needs to be `precache_query` instead of `qeury`

Reply:
I think this is OK. `query` is the name of the missing param. `precache_query` is the name of the task.

Reply:
Fixed.